specification/communications-manager.yaml

```diff
@@ -4,34 +4,34 @@ info:
   title: Communications Manager API
   description:  |
     ## Overview
-    'TBC'
+    TBC
 
     ## Who can use this API
-    'TBC'
+    TBC
 
     ## Related APIs
-    'TBC'
+    TBC
 
     ## API status and roadmap
-    'TBC'
+    TBC
 
     ## Service Level 
-    'TBC'
+    The Communications Manager API is currently not in an integration or production environment.\n
```

Comment:
Can remove \n from this line - \n is used to indicate that there is a new line (an enter) when it would otherwise not be visible.

Reply:
Looks good now, approving.

Reply:
Removed now and made the commit.